install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -126,11 +126,14 @@ from plug-in to plug-in.
 
 To set a _StorageClass_ as the cluster-wide default:
 ----
-   storageclass.beta.kubernetes.io/is-default-class: "true"
+   storageclass.kubernetes.io/is-default-class: "true"
 ----
 This enables any Persistent Volume Claim (PVC) that does not specify a specific
 volume to automatically be provisioned through the _default_ StorageClass
 
+Note that beta annotation `storageclass.beta.kubernetes.io/is-default-class` is still
```

Comment:
I suggest putting this in a note box:

    [NOTE]
    ====
    Beta annotation `storageclass.beta.kubernetes.io/is-default-class` is still working.  
    However, it will be removed in a  future release.
    ====